Storage/test.cc

```diff
@@ -213,7 +249,31 @@ TEST(TransactionalStorage, SmokeTest) {
       }).Go();
     }
 
-    // Rollback a transaction involving a `Matrix`.
+    // Iterate over the One2One
+    {
+      storage.Transaction([](MutableFields<Storage> fields) {
+        EXPECT_FALSE(fields.o.Empty());
+        std::multiset<std::string> data;
+        for (const auto& element : fields.o.Rows()) {
+          data.insert(current::ToString(current::storage::sfinae::GetRow(element)) + ',' +
+                      current::ToString(current::storage::sfinae::GetCol(element)) + '=' +
+                      current::ToString(element.phew));
+        }
+        EXPECT_EQ("1,one=1 3,too=4", current::strings::Join(data, ' '));
+      }).Go();
```

Comment:
Let's check the result of every transaction in the test. Because it could fail so that nobody will notice (as it were in `docu` test that you fixed) - `EXPECT_*` just won't be called. And yes, `docu` tests should be also fixed more thoroughly :)